nlg/utils.py

```diff
@@ -117,7 +117,7 @@ def __call__(self, func):
 
 def is_overlap(x, y):
     """Whether the token x is contained within any span in the sequence y."""
-    if "NUM" in [c.pos_ for c in x]:
+    if 'NUM' in [c.pos_ for c in x]:
         return False
     return any([x.text in yy for yy in y])
```

Comment:
Making an exception only for `NUM` or `CARDINAL` NEs is moot. Those should not reach this func at all.